examples/cloud_storage_stream_inference/README.md

```diff
@@ -0,0 +1,102 @@
+# Using fsspec to stream data from cloud storage providers for bacth inference
+
+This example shows how to use fsspec to stream large amount of data from cloud storage like s3, google cloud storage, azure cloud etc. and use it to create requests to torchserve for large scale batch inference with a large batch size.
```

Comment:
Please add a line or 2 on fsspec and add a link to their github page

Reply:
Added info about fsspec